gnmi_cli_py/py_gnmicli.py

```diff
@@ -465,46 +470,68 @@ def main():
   user = args['username']
   password = args['password']
   form = args['format']
+  create_connections = args['create_connections']
   paths = _parse_path(_path_names(xpath))
   kwargs = {'root_cert': root_cert, 'cert_chain': cert_chain,
             'private_key': private_key}
   certs = _open_certs(**kwargs)
   creds = _build_creds(target, port, get_cert, certs, notls)
-  stub = _create_stub(creds, target, port, host_override)
-  if mode == 'get':
-    print('Performing GetRequest, encoding=JSON_IETF', 'to', target,
-          ' with the following gNMI Path\n', '-'*25, '\n', paths)
-    response = _get(stub, paths, user, password, prefix)
-    print('The GetResponse is below\n' + '-'*25 + '\n')
-    if form == 'protobuff':
-      print(response)
-    elif response.notification[0].update[0].val.json_ietf_val:
-      print(json.dumps(json.loads(response.notification[0].update[0].val.
-                                  json_ietf_val), indent=2))
-    elif response.notification[0].update[0].val.string_val:
-      print(response.notification[0].update[0].val.string_val)
-    else:
-      print('JSON Format specified, but gNMI Response was not json_ietf_val')
-      print(response)
-  elif mode == 'set-update':
-    print('Performing SetRequest Update, encoding=JSON_IETF', ' to ', target,
-          ' with the following gNMI Path\n', '-'*25, '\n', paths, json_value)
-    response = _set(stub, paths, 'update', user, password, json_value)
-    print('The SetRequest response is below\n' + '-'*25 + '\n', response)
-  elif mode == 'set-replace':
-    print('Performing SetRequest Replace, encoding=JSON_IETF', ' to ', target,
-          ' with the following gNMI Path\n', '-'*25, '\n', paths)
-    response = _set(stub, paths, 'replace', user, password, json_value)
-    print('The SetRequest response is below\n' + '-'*25 + '\n', response)
-  elif mode == 'set-delete':
-    print('Performing SetRequest Delete, encoding=JSON_IETF', ' to ', target,
-          ' with the following gNMI Path\n', '-'*25, '\n', paths)
-    response = _set(stub, paths, 'delete', user, password, json_value)
-    print('The SetRequest response is below\n' + '-'*25 + '\n', response)
-  elif mode == 'subscribe':
-    request_iterator = gen_request(paths, args, prefix)
-    subscribe_start(stub, args, request_iterator)
+
+  while True:
+    if create_connections > 0:
+        create_connections -= 1
+    elif create_connections == 0:
+        break
+    elif create_connections < -1:
+        print('''
```

Comment:
>print [](http://example.com/codeflow?start=8&length=5)

Instead of manual check arg range and print error, you may leverage built-in capability in ArgumentParser. ref: [https://stackoverflow.com/a/25295717/2514803](https://stackoverflow.com/a/25295717/2514803)